server/mldb_server.cc

```diff
@@ -499,8 +499,8 @@ getPackageDocumentationPath(const Package & package) const
     // by any means guaranteed.
 
     if (package.packageName() == "builtin")
-        return "/doc/builtin/";
-    return "/v1/plugins/" + package.packageName() + "/doc/";
+        return httpBaseUrl + "/doc/builtin/";
+    return httpBaseUrl + "/v1/plugins/" + package.packageName() + "/doc/";
```

Comment:
I think it does on linux since url and path are more or less the same thing.  If we don't use boost::filesystem then we can simply trim trailing slashes.  That's the case I was trying to prevent.